### ISSUE TITLE
Add libwallaby sub module compilation into yarn build/yarn watch

### ISSUE THREAD

**Issue by bmcdorman** [kipr/Simulator#246]
I propose removing the manual compilation step for libwallaby entirely and integrating it into the frontend build workflow. It would be cool if we could “watch” it like we do the frontend, but probably unnecessary.

**Comment by navzam** [comment]
Is this worth doing if the compile endpoint is about to move away from the frontend and into its own service? Maybe libwallaby compilation could be integrated into the build workflow for that service?

**Comment by bmcdorman** [comment]
Yeah, fair enough. Once the compiler microservice is split out libwallaby won't be in this repo at all.
